app/Models/Node.php

```diff
@@ -307,9 +307,7 @@ public function systemInformation(): array
 
                 if ($message->startsWith('cURL error 6: Could not resolve host')) {
                     $message = str('Could not resolve host');
-                }
-
-                if ($message->startsWith('cURL error 28: Failed to connect to ')) {
+                } elseif ($message->startsWith('cURL error 28: Failed to connect to ')) {
```

Comment:
You're triggering me, this was fine before

```suggestion
                }
                
                if ($message->startsWith('cURL error 28: Failed to connect to ')) {
```